### ISSUE TITLE
Make schema name configurable when in query mode

### ISSUE THREAD

**Issue by JeroenSoeters** [confluentinc/kafka-connect-jdbc#90]
Currently when the JDBC connector is running in query mode the schema name defaults to null which causes the AvroConverter to fill in ConnectDefault. I would suggest adding a configuration property query.schema.name for the connector to be able to change this.

**Comment by shikhar** [comment]
Might make sense to fix this as part of #123 / #124

**Comment by boyenn** [comment]
Was this ever fixed?

**Comment by ewencp** [comment]
I don't think this is fixed yet, but a more general solution should be coming with https://cwiki.apache.org/confluence/display/KAFKA/KIP-66%3A+Single+Message+Transforms+for+Kafka+Connect (assuming it is voted in). This would work for all connectors rather than requiring every connector developer to implement something similar.

**Comment by wicknicks** [comment]
Closing this since the SMT workaround has solved the problem for multiple people (#123).
